app/src/full/java/io/homeassistant/companion/android/sensors/AndroidAutoSensorManager.kt

```diff
@@ -25,29 +46,112 @@ class AndroidAutoSensorManager : SensorManager, Observer<Int> {
             "mdi:car",
             updateType = SensorManager.BasicSensor.UpdateType.INTENT
         )
+        private val fuelLevel = SensorManager.BasicSensor(
+            "android_auto_fuel_level",
```

Comment:
Given that these new sensors are also for Android Automotive I wonder if we should consider a different naming approach here? I understand we already have the connection sensor up but that does not really matter for Android Automotive as the phone may not be connected. Not really sure what makes the most sense here so thought I would spark the convo :)

Reply:
I kinda agree but I'm not really sure of what to use as a prefix that would make sense for both

Reply:
Yea we started to have a small discussion on this. This has the potential of creating a lot more sensor entities which we obviously don't want to say no to but we should make sure the sensor management organization is good. Once we merge with the IDs we won't be able to change that without causing a breaking change for users.

@jpelgrom / @JBassett  what do you guys think? Should we keep all the android auto/automotive sensors together? should we consider another naming convention so we dont have so many added in one area? a lot of things to consider here so we dont introduce a breaking change later.

Reply:
Split up because Automotive has a minimal flavor while Android Auto sensors will only ever be for the full flavor.

Reply:
I like `car` and `vehicle` prefixes actually. Good point about it being split up due to the automotive minimal flavor. So these should be added in either `main` or `common` then correct?